databuilder/README.md

```diff
@@ -1270,6 +1270,114 @@ job = DefaultJob(conf=job_config,
 job.launch()
 ```
 
+### [ElasticsearchMetadataExtractor](./databuilder/extractor/es_metadata_extractor.py)
+
+The included `ElasticsearchMetadataExtractor` provides support for extracting basic metadata for Elasticsearch indexes.
+
+It extracts index metadata into `TableMetadata` model so the results are retrievable the same way as table metadata.
```

Comment:
any reason we are using TableMetadata model here? Shouldn't this be a new model for this use case?

Reply:
Hm I think TableMetadata is a very diverse concept and can be applied on NoSQL DBs as well. Unless we would want a different type of resource in UI for those but I am not sure how would it differ from regular table view and whether it makes sense to make such distinction.